### PR TITLE
Ext tests

### PR DESCRIPTION
**Pull request by ibragins**
Fixed some errors, now tests should pass normally

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.scenario.multi.nic.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// import { withResource } from '@console/shared/src/test-utils/utils';
```
Comment (rhrazdil) [inline frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.scenario.multi.nic.ts]: Please remove the commented lines

File: frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.scenario.multi.nic.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { V2V_VM_IMPORT_TIMEOUT, VM_STATUS } from '../tests/utils/consts';
+import { VirtualMachine } from '../tests/models/virtualMachine';
+import { vmwareVMMultiNicConfig } from './v2v.configs';
+import {
+  // removeLeakedResources,
```
Comment (rhrazdil) [inline frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.scenario.multi.nic.ts]: This one too

File: frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.configs.ts [hunk 3]
```diff
@@ -18,3 +18,20 @@ export const vmwareVMConfig: VMImportConfig = {
...
+    hostname: V2V_INSTANCE_HOSTNAME,
+    username: V2V_INSTANCE_USERNAME,
+    password: V2V_INSTANCE_PASSWORD,
+    saveInstance: true,
+  },
+  operatingSystem: OperatingSystem.RHEL7_6,
+  workloadProfile: WorkloadProfile.SERVER,
+};
+
+export const importConfig = [vmwareVMConfig, vmwareVMMultiNicConfig];
```
Comment (rhrazdil) [inline frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.configs.ts]: `v2vImportConfigs` may be more suitable name.

File: frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.scenario.multi.nic.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+  beforeAll(async () => {
+    createResources([multusNAD]);
+  });
+
+  afterAll(async () => {
+    deleteResources([multusNAD]);
+  });
+
+  it(
```
Comment (rhrazdil) [inline frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.scenario.multi.nic.ts]: So this creates only the one VM. You want to loop over the list of VM configs you created.

```
v2vImportConfigs.forEach( (importConfig) => {
  it(
    `Imports VM ${importConfig.name} from VMware Instance`,
    async () => {
      const vm = new VirtualMachine({ name: importConfig.name, namespace: testName });
      await withResource(leakedResources, vm.asResource(), async () => {
        await vm.import(importConfig);
        await vm.waitForStatus(VM_STATUS.Off, V2V_VM_IMPORT_TIMEOUT);
      });
    },
    V2V_VM_IMPORT_TIMEOUT,
  );
})
```

File: frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.scenario.multi.nic.ts (hunk 1 above)
Comment (ibragins) [reply frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.scenario.multi.nic.ts]: Fixed

File: frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.scenario.multi.nic.ts (hunk 2 above)
Comment (ibragins) [reply frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.scenario.multi.nic.ts]: Fixed

File: frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.configs.ts (hunk 3 above)
Comment (ibragins) [reply frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.configs.ts]: Fixed

File: frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.scenario.multi.nic.ts (hunk 4 above)
Comment (ibragins) [reply frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.import.scenario.multi.nic.ts]: I would like not to build a loop that you described, cause basic test should be used as a sanity test, each time when new build is automatically tested. Test with 2 NICs - it is advanced and should be executed in a suite. I don't want them to be executed together

**Pull request closed without merging**